### PR TITLE
misc: Add resources from OCW

### PR DESCRIPTION
**Pull request by StefanJum**
Add resources for the Operating Systems course.

### CODE REVIEW

File: content/misc/resources.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+There are no classes / labs in the 27 November 2023 - 1 December 2023 week.
+
+You can find the full calendar in multiple formats below:
+
+- Calendar ID: `d2biu4r5gcv83ndamn6tpe0onc@group.calendar.google.com`
+- [XML](http://www.google.com/calendar/feeds/d2biu4r5gcv83ndamn6tpe0onc%40group.calendar.google.com/public/basic)
+- [ICAL](http://www.google.com/calendar/ical/d2biu4r5gcv83ndamn6tpe0onc%40group.calendar.google.com/public/basic.ics)
+- [HTML](http://www.google.com/calendar/embed?src=d2biu4r5gcv83ndamn6tpe0onc%40group.calendar.google.com&ctz=Europe/Bucharest)
+
+You can also find the course planning [here](https://docs.google.com/spreadsheets/d/11-csi2HtJP1HMT63NWy0Y9WpHxyVpxN-hqw7Y45ZH5c/edit?usp=sharing).
```
Comment (Alex-deVis) [inline content/misc/resources.md:30]:
```suggestion
You can also find the course planning [here](https://docs.google.com/spreadsheets/d/1HVCP7ybjrtrtN5MMQYbSnvHoals5IbagtY1SaG9K3FE/edit?usp=sharing).
```

File: content/misc/resources.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+You can import the `.ova` file in [VirtualBox](https://www.virtualbox.org/) or [VMware](https://www.vmware.com/).
+Follow the instructions on the official websites for installation.
+
+### UTM (macOS >= 11)
+
+If you are using an `M1` Apple system, you will not be able to run the virtual machine using VirtualBox or VMware.
+You will need to use [`UTM`](https://mac.getutm.app/), along with a [`.qcow2`](https://repository.grid.pub.ro/cs/so/linux-2024/SO-Ubuntu-22-04-03-LTS.utm.zip) image.
+You will need to log in using your `UPB` account.
+
+After you install `UTM` and download and unzip the archive, you can import it using the `Open existing VM` option in `UTM`.
```
Comment (Alex-deVis) [inline content/misc/resources.md:51]: You can also reference the instructions from [macos-vm/README.md](https://github.com/cs-pub-ro/operating-systems/blob/main/util/macos-vm/README.md).

File: content/misc/resources.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+List of resources:
+
+- [GitHub Repository](https://github.com/cs-pub-ro/operating-systems)
+- [Moodle Class](https://curs.upb.ro/2023/course/view.php?id=7774) (used for homework submissions, quizzes, announcements, etc.)
+- [Teams Class](https://teams.microsoft.com/l/team/19%3aLTQa0tGvTM_L1c8N9F_dwZJ8RKLA_KqVejJJdeLfHos1%40thread.tacv2/conversations?groupId=ca6de354-95d1-4b60-9668-d79997e9d058&tenantId=2d8cc8ba-8dda-4334-9e5c-fac2092e9bac)
+- [Rules and Grading](https://cs-pub-ro.github.io/operating-systems/rules-and-grading)
+- [Books / Reading Materials](http://elf.cs.pub.ro/so/res/doc/) (you will need to log in using your `UPB` account)
+- [OS Calendar](https://calendar.google.com/calendar/u/0/embed?src=d2biu4r5gcv83ndamn6tpe0onc@group.calendar.google.com&ctz=Europe/Bucharest)
+
```
Comment (teodutu) [inline content/misc/resources.md:13]:
```suggestion
- [OS Activities Planning](https://docs.google.com/spreadsheets/d/1HVCP7ybjrtrtN5MMQYbSnvHoals5IbagtY1SaG9K3FE/edit)

```
So students know what lab is next.

**Review by teodutu — CHANGES_REQUESTED** [review]
Let's add the planning as well as in my inline suggestion.

**Review by Alex-deVis — APPROVED** [review]
Looks good.

**Pull request merged**